UPGRADE.asciidoc

```diff
@@ -68,6 +68,13 @@ Upgrading from a version prior to X.Y.Z
 
 Once completed, update the file /usr/local/pf/conf/currently-at to match the new release number (PacketFence X.Y.Z).
 
+Samba cache directory changed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Rejoining the domains from PacketFence GUI is required. 
```

Comment:
When we rejoin the domain, the configuration is regenerated. There is no need to refresh the domains for the configuration to be regenerated (and the cache path to be changed). rejoin does a leave and a join:
```
sub join_domain {
    my ($domain) = @_;
    my $logger = get_logger();
    my $chroot_path = chroot_path($domain);

    regenerate_configuration();

    my $info = $ConfigDomain{$domain};
[...]
```